src/main/java/jenkins/plugins/instana/HttpRequestExecution.java

```diff
@@ -48,8 +49,26 @@ static HttpRequestExecution from(ReleaseMarker http, TaskListener taskListener)
 		JSONObject jsonObject =  new JSONObject();
 		jsonObject.put("name", http.getReleaseName());
 		jsonObject.put("start", http.getReleaseStartTimestamp());
+		if (http.getServiceNames() != null) {
+			List<JSONObject> services = new ArrayList<>();
+			for (String serviceName : http.getServiceNames()) {
+				JSONObject service = new JSONObject();
+				service.put("name", serviceName);
+				services.add(service);
+			}
+			jsonObject.put("services", services);
+		}
+		if (http.getApplicationNames() != null) {
+			List<JSONObject> applications = new ArrayList<>();
+			for (String applicationName : http.getApplicationNames()) {
+				JSONObject service = new JSONObject();
+				service.put("name", applicationName);
+				applications.add(service);
+			}
+			jsonObject.put("applications", applications);
+		}
 		String body = jsonObject.toString();
-
+		System.out.println(body);
```

Comment:
I think we should drop this one

Reply:
Or is this used for the assertions? If yes, we should either look for an alternative or at least comment so we know why it's here :)

Reply:
Yes, we should remove it from here.